### PR TITLE
education page

### PR DESCRIPTION
**Pull request by andyjih**

### CODE REVIEW

File: pages/solutions/education.tsx [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+                        height={492}
+                      />
+                    </Tab.Pane>
+                  </Tab.Content>
+                </Col>
+              </Row>
+            </Tab.Container>
+          </div>
+
+          {/* On Small screens, show a grid */}
```
Comment (teallarson) [inline pages/solutions/education.tsx:322]: Small screen views refer to patients instead of teachers/students

File: pages/solutions/education.tsx [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+            </Col>
+          </Row>
+          <br />
+          <Row className="align-items-center">
+            <Col md={4}>
+              <h3 className="homepageBlueH3">Send to Destinations</h3>
+              <p>
+                Integrations make it easy to send the right data to each tool
+                you use. No code is necessary to change what data gets sent. You
+                control exactly what data gets sent.
```
Comment (teallarson) [inline pages/solutions/education.tsx]: Might be worth mentioning here or near the Integrations component below that we're constantly adding destinations.  I've seen a variety of destinations used in K-12 and higher ed that we may not support yet.

**Review by teallarson — CHANGES_REQUESTED** [review]
Looks great!  I left one thought and one fix.

**Review by teallarson — APPROVED** [review]

**Comment by teallarson** [comment]
I didn't see a footer link if it's supposed to be there too!

**Comment by andyjih** [comment]
> I didn't see a footer link if it's supposed to be there too!

Oops! Good call. Added!

**Pull request merged**